eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -336,8 +336,8 @@ object Graph {
 
   object RoutingHeuristics {
 
-    // Number of blocks in two months
-    val BLOCK_TIME_TWO_MONTHS = 8640
+    // Number of blocks in one year
+    val BLOCK_TIME_ONE_YEAR = 52560
```

Comment:
I would write the formula, not just the result, it's easier to check.

```suggestion
    val BLOCK_TIME_ONE_YEAR = 365 * 24 * 6
```

Reply:
Good idea, done in https://github.com/ACINQ/eclair/pull/1796/commits/5e668b22240a40118a66f38042763e2f622b20df